sunpycube/cube/NDCube.py

```diff
@@ -464,6 +464,9 @@ def explode_along_axis(self, axis=None):
         # if axis is None then set axis as common axis.
         if axis is None:
             axis = self.common_axis
+        elif self.common_axis is not None:
+            if self.common_axis is not axis:
```

Comment:
I think this should be ```!=``` rather than ```is not``` according to my understanding of the subtle difference between the two.  It's less likely to crash but shouldn't let the user do something they didn't mean to in this case.  @Cadair may have a comment here?